data/json/snippets/lab.json

```diff
@@ -26,6 +26,209 @@
+      },
+      {
+        "id": "dream_2",
+        "//": "I saw him and he smiled with galaxies",
+        "text": "\"We are seen.  It's not long until we will be touched.\"\n\n\"Touched by what, exactly?\"\n\n\"Touched.  Chewed.  Tasted.  Digested.  Processed.  Excreted.\"\n\n\"Ramin, focus, please.  What is doing all that?  Or rather, what do you feel will be doing those things?\"\n\n\"It's big.  Bigger than the Sun.  Smaller than an atom.\"\n\n\"And what does this large small thing want, beyond touching us?\"\n\n\"It just is.  It needs no reason.  It needs only to exist.  It needs only to expand.\"\n\n\"Do you want to talk about Immersion 132R?\"\n\n\"There's nothing to talk about.  I wasn't there.  It didn't touch me.  I didn't look.  It didn't look back.  It didn't smile with a million rotten worlds in its mouth.\"\n\n\"Have you been taking your medication?\"\n\n\"Like they would help.  I tried, Doc, I really did.  But the truth is, I like having these thoughts.  They give me a certanity, and I am thankful for knowing how it will all end.  That we all belong to something, and nothing can take us from it now.\""
+      },
+      {
+        "id": "dream_3",
+        "//": "Something Fungi this way comes",
+        "text": "I keep dreaming the same dream ever since Dr. Yin's final session.  The same one she described with that vivid enthusiasm, like she could see it in the waking world as well.  I didn't understand the beauty of it back then, so I just took my notes as usual when one of them vents about the horrible things we encourage them to feel good about -- it is what I'm getting paid for, after all.  Looking at the transcript now, its dryness feels blasphemous: \"connected branches of grey vines\" he writes, unable to imagine the transcendce of the Whole stretching across infinity, connecting large and small, familiar and unknowable; \"moldy fuzz in a positive connotation - rural childhood?\" like an armchair psychotherapist grasping at straws, as if the eternal fractal was anything less than divine.  I am grateful for her allowing me to dream the same dream on for her, I only wish it came to me before I hastily reported the session to the bloodhounds at Security.  I remember being terrified of something, but the feeling melted in the radiance of my first night's sleep.  I will need to seek out others of a similar mind, to connect with them, to show them what we can become with time, but I need to be more careful than she was."
```

Comment:
```suggestion
        "text": "I keep dreaming the same dream ever since Dr. Yin's final session.  The same one she described with that vivid enthusiasm, like she could see it in the waking world as well.  I didn't understand the beauty of it back then, so I just took my notes as usual when one of them vents about the horrible things we encourage them to feel good about -- it is what I'm getting paid for, after all.  Looking at the transcript now, its dryness feels blasphemous: \"connected branches of grey vines\" he writes, unable to imagine the transcendence of the Whole stretching across infinity, connecting large and small, familiar and unknowable; \"moldy fuzz in a positive connotation - rural childhood?\" like an armchair psychotherapist grasping at straws, as if the eternal fractal was anything less than divine.  I am grateful for her allowing me to dream the same dream on for her, I only wish it came to me before I hastily reported the session to the bloodhounds at Security.  I remember being terrified of something, but the feeling melted in the radiance of my first night's sleep.  I will need to seek out others of a similar mind, to connect with them, to show them what we can become with time, but I need to be more careful than she was."
```